pyproject.toml

```diff
@@ -21,16 +22,15 @@ dependencies = [
 # On a mac, install optional dependencies with `pip install '.[dev]'` (include the single quotes)
 [project.optional-dependencies]
 dev = [
+    "mypy", # Used for static type checking of files
+  "pre-commit", # Used to run checks before finalizing a git commit
```

Comment:
```suggestion
    "pre-commit", # Used to run checks before finalizing a git commit
```

Reply:
Another indentation thing--I'll go in to the template to check these over afterwards